lib/core/src/sdk.rs

```diff
@@ -234,10 +245,34 @@ impl LiquidSdk {
             receive_swap_handler,
             chain_swap_handler,
             buy_bitcoin_service,
+            external_input_parsers,
         });
         Ok(sdk)
     }
 
+    fn get_all_external_input_parsers(config: &Config) -> Option<Vec<ExternalInputParser>> {
```

Comment:
NIT: do we need Option here or we can use an empty vector in that case? Perhaps it simplify a bit the code.

Reply:
Yeah, it wasn't needed. Removed in [015823c](https://github.com/breez/breez-sdk-liquid/pull/614/commits/015823c8b5e1ed95db60c25ee86d34e4f267eea0).